Task-5-database-in-go/DBhandler/query.go

```diff
@@ -0,0 +1,81 @@
+package DBhandler
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/joho/godotenv"
+	_ "github.com/lib/pq"
+
+	"manav402/server/models"
+)
+
+// exporting db variable to user around the module
+type DB struct {
+	Dbptr *sql.DB
+}
+var db = DB{}
+// var DB *sql.DB
+var err error
+
+// connect db funtion to initialize DB variable and connect to postgres database
+func ConnectDB() (*DB,error) {
+	m, err := godotenv.Read(".env")
```

Comment:
godotenv.Read()
why need .env reading two times.

you already reading env in main.go manage from their.
don't read env multiple times.